styleguide/derek/pattern-components/images/image.ejs

```diff
@@ -0,0 +1 @@
+<img src="http://placekitten.com/300/400"/>
```

Comment:
You don't need this trailing slash here. 

Also, just to confirm, the image on the overview and the image on individual leader pages are completely different aspect ratios.  Are there going to be two distinct images per leader on the website?